### ISSUE TITLE
Panel-buttons are displayed for inappropriate kind of tab (Statistics)

### ISSUE THREAD

**Issue by ask9** [dbeaver/dbeaver#35818]
### Description

Panel-buttons are displayed for inappropriate kind of tab (Statistics) if the select-statement has been executed before update-statement.

- use dvdrental database
- use this code:

```
select * 
from rental;

update rental
set staff_id = staff_id;
```

- Please also check other kinds of tabs for this issue

https://github.com/user-attachments/assets/f567dd07-6e12-47cc-b557-089390336446

### DBeaver Version

Enterprise Edition 24.2.2-202409192322

### Operating System

Windows 10

### Database and driver

PostgreSQL 16
PostgreSQL JDBC Driver 42.7.4

**Comment by E1izabeth** [comment]
Yes, these panels are not working for Statistics tab. We shouldn't show them for this tab.
